Python/Product/Django/DjangoEditorFactory.cs

```diff
@@ -16,11 +16,7 @@
 
 using System;
 using System.Runtime.InteropServices;
-#if DEV16_OR_LATER
```

Comment:
does it mean we don't support less version than Dev16 or the package is just work for both? I wonder.

Reply:
The builds are specific to a version of VS.

We have the 15.9 branch which targets VS 2017.

The master branch only targets VS 2019.

Until VS 2019 was GA, it was useful to allow build/run master branch on VS 2017, but now that VS 2019 is officially out, we don't need to keep maintaining support for VS 2017 in master branch.